pkg/api/acknowledge.go

```diff
@@ -20,25 +20,26 @@
 package api
 
 import (
+	"errors"
 	"fmt"
-
 	"github.com/go-kit/kit/log/level"
 	"github.com/nlopes/slack"
+	"github.com/PagerDuty/go-pagerduty"
 	"github.com/sapcc/pulsar/pkg/clients"
 )
 
 // acknowledge will:
 // 1. open a slack thread noting the acknowledger (slack user) and time
 // 2. add an emoji to the original slack message with the alert to indicate it's being worked on
 // 3. search and acknowledge the corresponding incident in pagerduty to avoid further esacalation (call, etc.)
-func (a *API) acknowledge(message slack.InteractionCallback) error {
+func (a *API) acknowledge(message slack.InteractionCallback) (*pagerduty.ListIncidentsResponse, error) {
```

Comment:
I don't get why `*pagerduty.ListIncidentsResponse` is being returned (and being nil all the time?) all the way to [this](https://github.com/sapcc/pulsar/blob/dbe87aa236280ca467d93a6a55b7730227ec0513/pkg/api/core.go#L146) only to get finally ignored. We only care whether there's an error, no?

Reply:
correct -- we hadn't before, but with not using our fork and going back to the origin this comes in

Reply:
you were right - it's remioved